share/chruby/auto.sh

```diff
@@ -22,12 +23,13 @@ function chruby_auto() {
 		chruby_reset
 		unset RUBY_AUTO_VERSION
 	fi
+	: "$old_" # restore $_ (last argument of last command executed)
 }
 
 if [[ -n "$ZSH_VERSION" ]]; then
 	if [[ ! "$preexec_functions" == *chruby_auto* ]]; then
 		preexec_functions+=("chruby_auto")
```

Comment:
FYI, the save/restore `$_` also works in `zsh`, even though it is not required as `zsh` saves/restores automagically.